pkgs/desktops/gnome-3/extensions/default.nix

```diff
@@ -0,0 +1,25 @@
+{pkgs, ...}: let
```

Comment:
They the `...` and why the blanket `pkgs`?

You just need `lib` and `callPackage`.